templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -225,7 +225,7 @@ parameters:
 - description: Container Image to use for Ansible Service Broker in format of imagename:tag
   displayname: Ansible Service Broker Image
   name: BROKER_IMAGE
-  value: ansible-service-broker:latest
+  value: ansibleplaybookbundle/ansible-service-broker:latest
```

Comment:
I modified the broker logging to verify it was truly using the local build. So this works great.